_pages/docs/reports/may2021.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: PyHC project status, May 2021
+permalink: /docs/reports/may2021/
+exclude: true
+---
+
+We held the sixth, and third fully-virtual, PyHC meeting from Monday May 10th - Thursday, May 13th on the Zoom platform. Each day spanned two hours (9 - 11 AM MT). The meeting covered core project updates, as well as relevant tutorials, hackathons, and a discussion on the overall direction of PyHC. This meeting format, and the presented content, were overall well-received by the meeting attendees. Presentations, hackathon notes, zoom recordings, and all other meeting materials can be located via the meeting web page. Due to the constantly-evolving nature of COVID-19 and worldwide/national restrictions changing frequently, we decided to keep the meeting format for our fall 2021 meeting as a TBD. Lastly, we submitted a virtual-only PyHC poster session proposal for the hybrid AGU 2021 meeting. Should the session be accepted, the conveners will be Julie Barnum, Shawn Polson, Nick Murphy, and Jonathan Niehof.  
```

Comment:
- A comma could go between "Monday" and "May 10th" but no big deal
 - "zoom recordings" in the fifth sentence could use a capital "Zoom"
 - We could hyperlink to the meeting web page in the fifth sentence

Reply:
Will add in all suggested changes. Thanks!